### PR TITLE
Support PHP version 8.0

### PR DESCRIPTION
**Pull request by ausi**

### CODE REVIEW

**Comment by ausi** [comment]
All errors seem to be related to https://github.com/avalanche123/Imagine/pull/740

The warnings are because of PHPUnit 9. I think we should use `symfony/phpunit-bridge` to make it possible to test all PHP versions from 7.1 to 8.0:
> It allows to create tests that are compatible with multiple PHPUnit versions (because it provides polyfills for missing methods, namespaced aliases for non-namespaced classes, etc.).

**Comment by ausi** [comment]
> I think we should use `symfony/phpunit-bridge` to make it possible to test all PHP versions from 7.1 to 8.0

Done. Works perfectly, all deprecation warnings are gone and the new assertion methods are available on older PHP versions too. 🎉

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -85,10 +89,40 @@ jobs:
...
+
+            - name: Checkout
+              uses: actions/checkout@v2
+
+            - name: Install the dependencies
+              run: |
+                  composer self-update --2
+                  composer config --global github-oauth.github.com ${{ secrets.GITHUB_TOKEN }}
+                  composer config repositories.dev vcs https://github.com/ausi/Imagine
+                  php -r '$data = json_decode(file_get_contents(__DIR__."/composer.json"), true); $data["require"]["imagine/imagine"] = "dev-fix/php-8 as 1.2.99"; file_put_contents(__DIR__."/composer.json", json_encode($data));'
```
Comment (ausi) [inline .github/workflows/ci.yml:120]: Once avalanche123/Imagine#740 got merged we can remove these lines

**Comment by ausi** [comment]
I think this can be merged and I can release version 1.0.3 afterwards.

**Review by leofeyer — APPROVED** [review]

**Pull request merged**